src/Core/Scheduler/Scheduler.js

```diff
@@ -256,6 +274,29 @@ Scheduler.prototype.getProtocolProvider = function getProtocolProvider(protocol)
     return this.providers[protocol];
 };
 
+
+Scheduler.prototype.addFormatParser = function addFormatParser(parser) {
+    // eslint-disable-next-line no-console
```

Comment:
To not forget to remove the log before merging, you should remove this comment imho ;)